client/src/utilities/feedback.js

```diff
@@ -1,14 +1,13 @@
 /* eslint-disable no-undef, no-use-before-define, no-underscore-dangle */
 
-let feedbackInited;
 
 export const init = (state) => {
-  if (feedbackInited || !state.profile) {
+  if (!state || !state.profile) {
     return;
   }
-  feedbackInited = true;
   window.bugyard('setUser', {
     email: 'devops@akvo.org',
+    name: state.profile.sub || 'Unknown',
```

Comment:
`name` is not a required field, and we already passing an `id`

Reply:
the thing is that user details are not yet working and i was trying to pass another piece of data to understand the problem better, but no idea yet